client/layout/sidebar-v2/navigator/style.scss

```diff
@@ -0,0 +1,37 @@
+.sidebar-v2__navigator-sub-menu {
+	padding: 0 !important;
+
+	.components-navigator-back-button {
+		margin-block-end: 8px !important;
+		color: var(--studio-gray-90) !important;
+		font-size: rem(16px);
+		font-weight: 600;
+		justify-content: start;
+
+		&:focus {
+			box-shadow: none;
+		}
+	}
+}
+
+.sidebar-v2__navigator-group-description {
+	color: var(--studio-gray-50) !important;
+	font-size: rem(12px);
+	margin-block-end: 16px !important;
+	padding: 0 12px;
+}
+
+.sidebar-v2__menu-item {
+	color: var(--studio-gray-90) !important;
+	font-size: rem(12px);
+
+	&.is-active {
+		border-radius: 4px !important;
+		background: var(--studio-green-0) !important;
+		color: var(--studio-green-50) !important;
```

Comment:
For Jetpack-specific styling, we should override this from the consuming component instead of the base component.

Reply:
Makes sense. Moved it here: https://github.com/Automattic/wp-calypso/pull/83073/commits/e655f6f14d2a89da4708c2f088a6af10a76b9f59